activestorage/lib/active_storage.rb

```diff
@@ -65,6 +65,8 @@ module ActiveStorage
   mattr_accessor :replace_on_assign_to_many, default: false
   mattr_accessor :track_variants, default: false
 
+  mattr_accessor :proxy_urls_expire_in, default: 365.days
```

Comment:
What's the use-case for letting these expire? Versus just setting it to the very far-future, like we do with assets?